### PR TITLE
Modify footer

### LINKED ISSUE #29: MVP - Footer section
As a user
I want to have access a several quick links, such as "Past events", "Social" and other
So I can find another resources about the organization and event

## ACCs
- [ ] Present a link for each past event
  - So far we only have [WebMadeira 2016](http://webmadeira.herokuapp.com/)
- [ ] Present links for each social media
  - Youtube, Medium...
- [ ] Meet the [specs](https://app.zeplin.io/project/5b3be80249cddedaa2fe54ba/screen/5b41f0ea0b64db87652d4acc)

**Important note:** All these links must be provided by the API

### PR DESCRIPTION
**Pull request by Comum**
Closes #29

### CODE REVIEW

File: src/store/actions/types.js [hunk 1]
```diff
@@ -2,15 +2,22 @@ const FETCH_ORGANIZATION = 'ORGANIZATION_FETCHED'
...
 const FETCH_EVENTS_FULFILLED = 'EVENTS_FETCHED_FULFILLED'
 const FETCH_EVENTS_REJECTED = 'EVENTS_FETCHED_REJECTED'
+
 const FETCH_EVENT_DETAILS = 'EVENT_DETAILS_FETCHED'
-const FETCH_EVENT_DETAILS_PENDING = 'EVENT_FETCHED_PENDING'
+const FETCH_EVENT_DETAILS_PENDING = 'EVENT_DETAILS_FETCHED_PENDING'
 const FETCH_EVENT_DETAILS_FULFILLED = 'EVENT_DETAILS_FETCHED_FULFILLED'
 const FETCH_EVENT_DETAILS_REJECTED = 'EVENT_DETAILS_FETCHED_REJECTED'
 
+const FETCH_PAST_EVENTS = 'PAST_EVENTS_FETCHED'
```
Comment (OrencioRodolfo) [inline src/store/actions/types.js]: Shouldn't this be `const FETCH_PAST_EVENTS = 'FETCH_PAST_EVENTS'`?

I know all the above types follow this approach, but I've changed it at #30

File: src/assets/img/icon-facebook.svg [hunk 2]
```diff
@@ -0,0 +1,12 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="9432.721 3904.3 33.671 33.7">
```
Comment (OrencioRodolfo) [inline src/assets/img/icon-facebook.svg]: I would rather create a `assets/icons` folder and place these files there without the `icon-` prefix

File: src/scenes/components/Footer/Footer.js [hunk 3]
```diff
@@ -1,15 +1,150 @@
...
+const SocialItem = props => (
+  <SocialListItem>
+    <a href={props.path}>
+      <img src={socialIcons[props.name]} alt={props.name} />
+    </a>
+  </SocialListItem>
 )
 
+const Footer = ({ pastEvents, social }) => {
+  const pastEventsList = pastEvents.map(pastEvent => (
```
Comment (OrencioRodolfo) [inline src/scenes/components/Footer/Footer.js]: I think we should move this to an isolated function outside the `render` function, receiving the `pastEvents` array

File: src/scenes/components/Footer/Footer.js [hunk 4]
```diff
@@ -1,15 +1,150 @@
...
+    <PastEventItem
+      key={pastEvent.id}
+      name={pastEvent.name}
+      path={pastEvent.path}
+    />
+  ))
+  const socialList = social.map(socialItem => (
```
Comment (OrencioRodolfo) [inline src/scenes/components/Footer/Footer.js]: I think we should move this to an isolated function outside the render function, receiving the social array

File: src/scenes/components/Footer/Footer.js [hunk 5]
```diff
@@ -1,15 +1,150 @@
...
+const PastEventItem = props => (
+  <WebConferencesListItem>
+    <WebConferenceLink href={props.path} target="_blank">{props.name}</WebConferenceLink>
+  </WebConferencesListItem>
+)
+
+const SocialItem = props => (
+  <SocialListItem>
+    <a href={props.path}>
+      <img src={socialIcons[props.name]} alt={props.name} />
```
Comment (OrencioRodolfo) [inline src/scenes/components/Footer/Footer.js]: Note: We should talk to the design for exporting the SVG files with a single path. This way we could create an `Icon` component and follow an approach similar to [Icons as React Components](https://medium.com/@david.gilbertson/icons-as-react-components-de3e33cb8792)

File: src/store/actions/types.js (hunk 1 above)
Comment (Comum) [reply src/store/actions/types.js]: 👍

File: src/assets/img/icon-facebook.svg (hunk 2 above)
Comment (Comum) [reply src/assets/img/icon-facebook.svg]: 👍

**Pull request merged**